### ISSUE TITLE
[bug] Meter report errors

### ISSUE THREAD

**Issue by blhoward2** [zwave-js/zwave-js#1316]
**Describe the bug**
All of my devices that check-in with energy reports throw an error under the node-zwave-js master branch with the zwavejs2mqtt fix#23 branch. 

### Master Branch

```
2021-01-08 03:42:44.407 ERROR GATEWAY: Error while discovering value 103-50-0-deltaTime-65537 of node 103: The config has not been loaded yet! The config has not been loaded yet!

ZWaveError: The config has not been loaded yet!

    at lookupMeter (/usr/src/app/node_modules/@zwave-js/config/src/Meters.ts:80:9)

    at Object.meterType (/usr/src/app/lib/Constants.js:21:19)

    at Gateway.discoverValue (/usr/src/app/lib/Gateway.js:1598:32)

    at Gateway.onValueChanged (/usr/src/app/lib/Gateway.js:234:10)

    at ZwaveClient.emit (events.js:314:20)

    at ZwaveClient.EventEmitter.emit (domain.js:483:12)

    at ZwaveClient.updateValue (/usr/src/app/lib/ZwaveClient.js:742:12)

    at ZwaveClient.onNodeValueUpdated (/usr/src/app/lib/ZwaveClient.js:390:15)

    at ZWaveNode.emit (events.js:314:20)

    at ZWaveNode.EventEmitter.emit (domain.js:483:12)

    at ZWaveNode.translateValueEvent (/usr/src/app/node_modules/zwave-js/src/lib/node/Node.ts:352:8)

    at ValueDB.emit (events.js:314:20)

    at ValueDB.EventEmitter.emit (domain.js:483:12)

    at ValueDB.setValue (/usr/src/app/node_modules/zwave-js/node_modules/@zwave-js/core/src/values/ValueDB.ts:224:10)

    at MeterCCReport.persistValues (/usr/src/app/node_modules/zwave-js/src/lib/commandclass/MeterCC.ts:602:12)

    at new MeterCCReport (/usr/src/app/node_modules/zwave-js/src/lib/commandclass/MeterCC.ts:534:8)
```

I get the same error message for all three values.

### Under the zwavejs2mqtt-dev image

```
2021-01-07 23:22:14.668 INFO ZWAVE: Node 103: value updated: 50-0-value-65537 1.506 => 1.506

2021-01-07 23:22:14.671 INFO ZWAVE: Node 103: value updated: 50-0-previousValue-65537 1.506 => 1.506

2021-01-07 23:22:14.675 INFO ZWAVE: Node 103: value updated: 50-0-deltaTime-65537 30 => 30
```

```
23:22:44.669 SERIAL « 0x0114000400670e32022164000005e2001e000005e2ed                      (22 bytes)
23:22:44.671 CNTRLR   [Node 103] [~] [Meter] value[65537]: 1.506 => 1.506               [Endpoint 0]
23:22:44.675 CNTRLR   [Node 103] [~] [Meter] previousValue[65537]: 1.506 => 1.506       [Endpoint 0]
23:22:44.680 CNTRLR   [Node 103] [~] [Meter] deltaTime[65537]: 30 => 30                 [Endpoint 0]
23:22:44.684 SERIAL » [ACK]                                                                   (0x06)
23:22:44.689 DRIVER « [Node 103] [REQ] [ApplicationCommand]
                      └─[MeterCCReport]
                          type:        Electric
                          scale:       kWh
                          rate type:   Consumed
                          value:       1.506
                          time delta:  30 seconds
                          prev. value: 1.506
```

**To Reproduce**

**Device information**
I'm see this on multiple devices including Aeotec Switch 6s, Aeon Nano Dimmers, and an Aeon Microdimmer.

**Logfile:** 

I can't generate better logfiles because logging does not work as described in the fix#23 PR.

**Comment by AlCalzone** [comment]
What's the difference to the zwavejs2mqtt-dev image?

You could try and run zwavejs2mqtt without Docker, to make sure the build process of that isn't the problem.

**Comment by blhoward2** [comment]
I can try that tonight.

**Comment by jcam** [comment]
Hmm, I'm getting this on my bridge controller branch too. After a half dozen flips between on/off, it got an unhandled exception (with the same GATEWAY error) and the whole server shut down

**Comment by AlCalzone** [comment]
I'm like 99% certain this happens due to the faulty docker image. This is **just** zwave-js:

![grafik](https://user-images.githubusercontent.com/17641229/104078430-44b2d800-521e-11eb-9b53-cdf35cf53985.png)

**Comment by blhoward2** [comment]
So what’s breaking the image then? It’s in every image I’ve built off fix#23/master.

**Comment by AlCalzone** [comment]
My guess is this https://github.com/zwave-js/zwavejs2mqtt/pull/181 but I'm not sure.

**Comment by blhoward2** [comment]
@AlCalzone I do not get this error in a container built with both https://github.com/zwave-js/zwavejs2mqtt/pull/181 and https://github.com/zwave-js/zwavejs2mqtt/pull/84. Since it exists when built with only fix#23, https://github.com/zwave-js/zwavejs2mqtt/pull/181 must fix it.

**Comment by blhoward2** [comment]
I’ve also confirmed that this is fixed when https://github.com/zwave-js/zwavejs2mqtt/pull/181 is built with the config directory properly linked (per my comment on that PR). As such, this seems to be an artifact.
